public/Lab-Files/Lesson02/Assignments/resume.html

```diff
@@ -0,0 +1,63 @@
+        </p>
+       
+    <h3>Highlights</h3>
+        <ul>
+            <li>Self-Directed</li>
+            <li>Orgainzed</li>
+            <li>Problem Solver</li>
+            <li>Resourceful</li>
+            <li>Reliable</li>
+        </ul>
+
+    <h3>Education</h3>
+        <ul>
+            <li>Current - Full Stack Web Developer Certification - San Antonio Coding Academy - San Antonio, TX</li>
+            <li>2009 - Bachelor of Arts: Psychology - University of Texas Pan American, Edinburg, TX</li>
+            <li>2005 - High School Diploma - McAllen High School - McAllen, TX</li>
+        </ul>
+
+    <h3>Work Experince</h3>
+        <div>02/2011 to Present</div>
+            <p>Chief Workshop Manager at Build A Bear Workshop - La Cantera - San Antonio, TX</p>
+                <p>I am accountable for the overall success of the workshop. Direct and lead a team of dynamic individuals to achieve the company objectives in regards to a WOW guest and associate ex- perience, meeting sales, and profitability goals set by the company.
+                Networking to grow party and overall business. Recruit, interview, and train top talent to be able to promote from within when positions open and to achieve successful results in guest service and sales. I confronts and addresses performance
+                issues or conflicts in a timely and consistent manner. Complete daily procedures, Maintain a daily awareness of merchandise in stock, current promotions, maintain impeccable housekeeping, maintenance, and maintain a positive and motivational
+                style in all communications and actions to promote productivity and ownership.</p>
+        <div>12/2016 to 05/2010</div>
+            <p>Assistant Store Director at Hollywood Video - McAllen, TX</p>
+                <p>Addressed customer inquiries and resolved complaints, assisted in recruiting and training new staff, and follow companies inventory policies.Delivered excellent customer service by greeting, and assisting each customer, open/close store, count
+                registers and deposits, dropped off deposits at bank, morning/nightly paperwork, and finish any projects that needed to be done in a timely manner.</p>
+        <div>05/2005 to 07/2006</div>
+            <p>Stock Associate at Victoria's Secret - McAllen, TX</p>
+                <p>Process shipment by the designated time allotted, Restock floor to company standers, keep   stockroom clean and organized, assist in floor sets, and help guest when out on the floor   with what they are looking for.</p>
+        
+
+    <h3>Work Accomplishments</h3>
+        <ul>
+            <li>2016-New Store Opener for Build A Bear Workshop</li>
+            <li>2015-Worked on writing training modules with HR team for Build A Bear Workshop</li>
+            <li>2013 to Present- Market Trainier for Build A Bear Workshop</li>
+        </ul>
+
+    <h3>References</h3>
+    <p>Given upon request</p>
+    
```

Comment:
Awesome job. It looks clean and organized.

Starting line 37, I noticed you wrapped only the dates inside a ```<div>```. I think it would be better to put the entire content, including the paragraph and heading, inside of it. Remember, HTML is only supposed to structure your document. You use CSS for any styling.